### DISCUSSION TITLE
Home Assistant considers icon_screen as 'unkown action'

### DISCUSSION THREAD

**Discussion by clfberlin** [Blackymas/NSPanel_HA_Blueprint#2337] [Q&A]
Hi there. I am a bit confused, because I keep getting the warning in my devices overview that my automizations use icon_screen, which is an 'unknown action', eventhough it works.
Thanks and cheers

**Comment by edwardtfn** [comment]
We don't have any `icon_screen` action with this project. Are you sure this is related to this project?
Which versions are you running?

**Reply by clfberlin** [reply]
Argh. Indeed wrong project. I apologize

**Reply by edwardtfn** [reply]
No problem... 🤣
